### DISCUSSION TITLE
How do you resume logging

### DISCUSSION THREAD

**Discussion by pkuehnel** [tobiasehlert/teslamateapi#198] [Q&A] [answered]
How exactly do you get TeslaMate to Resume logging?

**Comment by tobiasehlert** [comment]
Hi @pkuehnel,

The TeslaMateApi doesn't have anything to do with the logging from Tesla to TeslaMate. TeslaMateApi basically only extracts data from PostgreSQL by queries or by connecting to the MQTT server where data is populated by TeslaMate.
Or am I misunderstanding your question maybe?

Regards,
Tobias

**Reply by pkuehnel** [reply]
Yes, my question is, what does TeslaMateApi do, when the URL
`{_teslaMateBaseUrl}/api/v1/cars/{carId}/logging/resume` is called? It resumes the logging in TeslaMate, yes, but how does this work? Do you call a specific rest endpoint on the TeslaMate container?

**Reply by LelandSindt** [reply]
If I remember correctly this was added. via https://github.com/tobiasehlert/teslamateapi/blob/9bf4da1691cfe043e7132d93a8f9195f442c8716/CHANGELOG.md#added-5

and relates to this conversation.. 
https://github.com/adriankumpf/teslamate/discussions/1433

**Comment by pkuehnel** [comment, marked as answer]
It is `PUT /api/car/1/logging/resume`
